src/react/databrowser/buckets/details/__tests__/Overview.test.jsx

```diff
@@ -0,0 +1,135 @@
+import * as T from '../../../../ui-elements/TableKeyValue2';
+import * as actions from '../../../../actions/s3bucket';
+import { Button, Toggle } from '@scality/core-ui';
+import { BUCKET_INFO_RESPONSE } from '../../../../actions/__tests__/utils/testUtil';
```

Comment:
Please get `bucketInfoResponse` from '../../../../../js/mock/S3Client'
`BUCKET_INFO_RESPONSE` is specific to action testing, and we avoid using it elsewhere.

Reply:
done